style.css

```diff
@@ -10,13 +10,15 @@ body {
 }
 .header {
   display: flex;
-  justify-content: space-between;  
+  justify-content: space-between;
+  margin-bottom: 100px;  
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/HTML_CSS_2.0-Artem1998/pullRequest?prid=6242157)